data/src/main/java/me/androidbox/data/local/dao/AttendeeDao.kt

```diff
@@ -0,0 +1,21 @@
+package me.androidbox.data.local.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import me.androidbox.data.local.DatabaseConstant
+import me.androidbox.data.local.entity.AttendeeEntity
+
+@Dao
+interface AttendeeDao {
+    @Query("SELECT * FROM ${DatabaseConstant.ATTENDEE_TABLE}")
+    fun getAttendee(): Flow<List<AttendeeEntity>>
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertAttendee(attendeeModel: AttendeeEntity)
+
+    @Delete
+    fun deleteAttendee(attendeeModel: AttendeeEntity)
+
+    @Query("DELETE FROM ${DatabaseConstant.ATTENDEE_TABLE}")
+    fun deleteAllAttendee()
```

Comment:
I'd say you rather need a function to delete all attendees part of an event, but not all of them right? 🤔

Reply:
Just to clarify

You never want to delete ALL attendees from the DB, since the attendees are linked to different events
What you want, is to remove attendees from an event, but not from the DB

Reply:
So this will be a query to delete all the attendee that have been added to that event.

Reply:
If you save attendees as JSON part of an event, this whole DAO isn't necessary.

If you save attendees in a separate table and therefore need a DAO, yes it would be a query to delete all attendees part of an event with a given ID